pkg/cli/builder/table/writer.go

```diff
@@ -230,6 +230,10 @@ func (t *writer) writeDataObject(objs ...any) error {
 	return errors.Join(t.errs...)
 }
 
+type ObjectList struct {
```

Comment:
nit: can this be unexported if it is only going to be used in this file?

Reply:
Yup. Done!